### PR TITLE
navbar +homepage

### PR DESCRIPTION
**Pull request by piyushsh25**
1. Created navbar and footer as a component.
2. Added home page.

### CODE REVIEW

File: src/pages/HomePage/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import "../../styles/Categories.css"
+export const Categories=()=>{
+    return(
+        <div className="shopping-section">
```
Comment (Tanay-J) [inline src/pages/HomePage/Categories.jsx:4]: Categories need to be fetched from backend using api call

**Review by Tanay-J — COMMENTED** [review]
Suggestions:
- Folder structure can be improved. Try to keep all components in one folder and pages/screens in another.
- Include an index.js file to export all the components in a folder from a single file
- May include css modules

File: src/pages/HomePage/Categories.jsx (hunk 1 above)
Comment (piyushsh25) [reply src/pages/HomePage/Categories.jsx]: Made changes to the folder structure, used index js in all folders for export. Categories from backend using API call done... Please re-review.

Comment (Tanay-J) [reply src/pages/HomePage/Categories.jsx]: Approved

**Review by Tanay-J — APPROVED** [review]

**Review by GuptaEesh — COMMENTED** [review]
Code looks structured. good naming conventions used. Good to merge

**Pull request merged**